__test__/components/__snapshots__/card.test.js.snap

```diff
@@ -2,6 +2,14 @@
 
 exports[`Card Component renders without crashing 1`] = `
 <View
+  accessible={true}
```

Comment:
@ssanusi `__test__/components/__snapshots__/card.test.js.snap` it would be better to put this kind of files in `.gitignore` file, it will allow to only push relevant work!